packages/cli/package.json

```diff
@@ -28,14 +28,14 @@
     "test": "export TZ=UTC && jest --ci --silent"
   },
   "dependencies": {
-    "@celo/walletkit": "^0.0.14",
+    "@celo/utils": "^0.0.6-beta5",
+    "@celo/contractkit": "^0.1.1",
     "@oclif/command": "^1",
     "@oclif/config": "^1",
     "@oclif/plugin-help": "^2",
     "ethereumjs-util": "^5.2.0",
     "bip32": "^1.0.2",
     "bip39": "^2.5.0",
-    "bn.js": "^5.0.0",
```

Comment:
So, mobile uses BigNumber, the only other project that uses BN is protocol, and that is because web3 1.xx-beta uses BN by default. But it was mentioned to me, that we should stick to BigNumber for now, and then decide if we move to BN

So, actually BigNumber is/was the default.

Reply:
Ah. I guess I was assuming that maybe this change meant that there was a decision to *not* do BN anywhere. You're saying it's still on the table for protocol?